spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/java/org/springframework/cloud/data/module/deployer/cloudfoundry/CloudFoundryClient.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
These cloudfoundry REST client classes (and the Request/Response pairs) are essentially copied from the `cf-java-client` code (soon to be in v3). Eventually we ought to use that code directly, but at the moment it is hard to use it without forcing us to be a Web Application (I think). Also `cf-java-client` looks much too heavyweight for our purposes.